cumulusci/cli/runtime.py

```diff
@@ -98,23 +98,20 @@ def get_org(self, org_name=None, fail_if_missing=True):
     def check_org_expired(self, org_name, org_config):
         if org_config.scratch and org_config.date_created and org_config.expired:
             click.echo(click.style("The scratch org is expired", fg="yellow"))
-            if click.confirm("Attempt to recreate the scratch org?", default=True):
-                self.keychain.create_scratch_org(
-                    org_name,
-                    org_config.config_name,
-                    days=org_config.days,
-                    set_password=org_config.set_password,
-                )
-                click.echo(
-                    "Org config was refreshed, attempting to recreate scratch org"
-                )
-                org_config = self.keychain.get_org(org_name)
-                org_config.create_org()
-            else:
-                raise click.ClickException(
-                    f"The target scratch org is expired.  You can use cci org remove {org_name} "
-                    "to remove the org and then recreate the config manually"
+            self.keychain.create_scratch_org(
+                org_name,
+                org_config.config_name,
+                days=org_config.days,
+                set_password=org_config.set_password,
+            )
+            click.echo(
+                click.style(
+                    "Org config was refreshed, attempting to recreate scratch org",
```

Comment:
Let's move this above the call to create_scratch_org so it's shown even if scratch org creation fails. And let's change it to "Attempting to recreate expired scratch org." I don't think we have reason to conclude that the org config was refreshed.

Reply:
never mind, I forgot create_scratch_org just overwrites the keychain entry and it doesn't actually get created until we call create_org